apps/web/src/components/Shared/Login/New/Pending.tsx

```diff
@@ -15,6 +29,78 @@ const Pending: FC<PendingProps> = ({ handle, txHash }) => {
     variables: { request: { txHash } },
     pollInterval: 1000
   });
+  const { push } = useRouter();
+  const setShowAuthModal = useAuthStore((state) => state.setShowAuthModal);
+  const [authenticate] = useAuthenticateMutation();
+  const [getProfiles] = useUserProfilesLazyQuery();
+  const { address } = useAccount();
+
+  //get profiles with the address
+  useUserProfilesQuery({
+    variables: { ownedBy: address },
+    pollInterval: 1000
+  });
```

Comment:
what if we do this instead:
```suggestion
const { data: userProfilesData } = useUserProfilesQuery({
    variables: { ownedBy: address }
});
```

And lower we could then do this:
```
if (!userProfilesData?.profiles?.items?.length) {
        keepModal = true;
        console.error('No profiles with this address');
      } else {
        const profiles: any = userProfilesData?.profiles?.items
          ?.slice()
          ?.sort((a, b) => Number(a.id) - Number(b.id))
          ?.sort((a, b) => (a.isDefault === b.isDefault ? 0 : a.isDefault ? -1 : 1));

        const currentProfile = profiles[0];
        setProfiles(profiles);
        setCurrentProfile(currentProfile);
        setProfileId(currentProfile.id);
        keepModal = false;
      }
```

Then we would not need to call `getProfiles()` (saving one XHR call)

Reply:
Well done! I had to let ` pollInterval: 1000` to make it work!!!
lets test it harder!!

Reply:
I'm surprised you had to keep the poll interval as `useUserProfilesQuery` should be executed each time the component is rendered 🤔